rebasehelper/application.py

```diff
@@ -217,12 +217,17 @@ def _initialize_data(self):
         self.old_rest_sources = [os.path.abspath(x) for x in self.spec_file.get_sources()[1:]]
         self.new_rest_sources = [os.path.abspath(x) for x in self.rebase_spec_file.get_sources()[1:]]
 
-    def get_rpm_packages(self, dirname):
-        """
-        Function returns RPM packages stored in dirname/old and dirname/new directories
+    def check_rpm_packages(self, dirname: str) -> bool:
+        """Checks if there are RPM packages in dirname/old-build and
+        dirname/new-build. Updates results store with the build data
+        found in the specified directory, if any.
```

Comment:
This is rather redundant. If you want to keep the first sentence, you might as well just add ` and updates results store accordingly` and drop the second one.